### ISSUE TITLE
Hide Icons from Screen Readers that Have Associated Text

### ISSUE THREAD

**Issue by mtribone** [samvera/hyrax#1258]
### Descriptive summary

In Hyrax 2.0.0 alpha we use icons with certain interface elements to enhance meaning that can be read aloud incorrectly and be a distraction for users of screen readers.

### Rationale

"Modern versions of assistive technology, like screen readers, will read CSS generated content (how Font Awesome icons are rendered), as well as specific Unicode characters. When reading our default markup for rendering icons, assistive technology may have the following problems."

- The assistive technology may not find any content to read out to a user
- The assistive technology may read the unicode equivalent, which could not match up to what the icon means in context, or worse is just plain confusing
- Many dyslexic people find it helpful to swap out a website’s typeface for something like OpenDyslexic. But icon fonts get replaced as well, which makes for a frustratingly broken experience.

http://fontawesome.io/accessibility/

As a first step to providing better accessibility, we can "hide" the font icons from the accessibility tree.

`aria-hidden="true"`

### Expected behavior

Visual elements that don't convey meaning and are decorative should not be read by assistive technology.

### Actual behavior

Icons are read aloud.

### Related work

https://github.com/psu-stewardship/scholarsphere/issues/511
https://github.com/psu-stewardship/scholarsphere/pull/559

**Comment by adamjarling** [comment]
These are coming from Blacklight?  

![catalog-index-blacklight-icons](https://user-images.githubusercontent.com/3020266/43729048-a31113c0-996c-11e8-98d6-8d55834b6c3e.png)
